service/flags.go

```diff
@@ -17,25 +17,41 @@ package service // import "go.opentelemetry.io/collector/service"
 import (
 	"errors"
 	"flag"
-	"strings"
-
+	"fmt"
+	"go.opentelemetry.io/collector/config"
```

Comment:
I wonder why the linter didn't catch this. The import order seems odd.

Reply:
I ran `gofmt -w .` on the project. This is how it looks:
![image](https://user-images.githubusercontent.com/59079323/196601747-23c55841-384f-47a1-b941-95620db42d2e.png)
Should I look up correct import order and implement changes manually if this is not in line?

Reply:
Run `make lint` and see if it complains. I expect it to complain with something like this:

```
service/flags.go:20: File is not `goimports`-ed with -local go.opentelemetry.io/collector (goimports)
	"fmt"
```

You can then run the following command to fix it automatically for you:

```console
make fmt
```